.github/workflows/build-dev.yml

```
@@ -0,0 +1,146 @@
+  workflow_dispatch:
+  push:
+    branches:
+      - 'dev'
+    paths:
+      - 'DEV-VERSION'
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)